services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -2249,7 +2257,14 @@
                   ]
                 },
                 "dashboardTitle": "Report on grievances by service for each plan",
-                "drawerTitle": "Report encounter data for",
+                "drawerTitle": "Report on grievances by service for",
+                "drawerReminder": "Complete all fields and select the Save & close button to save this section.",
+                "drawerInfo": [
+                  {
+                    "type": "p",
+                    "content": "A single grievance may be related to multiple service types and may therefore be counted in multiple categories."
+                  }
+                ],
```

Comment:
@braxex can we assume this doesn't need to have an ETL to apply retroactively? Kim caught this missing.

Reply:
As long as design is ok with it being only forward-looking, yes. We should update the version in `mcpar.json` if changes are being made though.

Reply:
got approval from both Megan and Kim that it's okay if this change is only foward-looking